exercises/perfectShuffle.js

```diff
@@ -9,6 +9,22 @@
 
 function perfectShuffle(array) {
   // This is your job. :)
+  let firstHalfDeck = [];
+  let secondHalfDeck = [];
+  let halfDeck = array.length/2;
+  let shuffledDeck = [];
+  for (let i = 0; i < array.length; i++) {
+    if (i < halfDeck){
+      firstHalfDeck.push(array[i])
+    } else {
+      secondHalfDeck.push(array[i]);
+    }
+  }
+  for(let j = 0; j < firstHalfDeck.length; j++) {
```

Comment:
You can use `i` here again. The `let` ensures it's not in the same scope as the previous `i`.